docs/source/control/grid/synchronization.rst

```diff
@@ -0,0 +1,62 @@
+Synchronization Methods
+=======================
+
+Phase-Locked Loop
+-----------------
+
+The :doc:`/grid_examples/grid_following/index` examples use a Phase-Locked Loop (PLL) to synchronize with the grid.
+The block diagram of the PLL is shown in the figure below.
+
+.. figure:: ../figs/pll.svg
+   :width: 100%
+   :align: center
+   :alt: Block diagram of the phase-locked loop for grid synchronization
+   :target: .
+
+   Block diagram of the phase-locked loop.
+
+The PLL drives the signal :math:`\hat{u}_\mathrm{gq}` to zero, leading to :math:`\hat{\vartheta}_\mathrm{g}=\vartheta_\mathrm{g}` 
+and :math:`\hat{u}_\mathrm{gd}=u_\mathrm{gd}` in ideal conditions. 
+The grid voltage-vector :math:`\boldsymbol{u}_\mathrm{g}^\mathrm{s}=u_\mathrm{g} \mathrm{e}^{\mathrm{j} \vartheta_\mathrm{g}}` is measured.
+The angle :math:`\vartheta_\mathrm{g}` can be noisy and it is not directly used in the control. 
+Instead, the PLL tracks :math:`\vartheta_{g}` and filters its noise and harmonics above the PLL bandwidth.
+
+The gain selection:
+
+.. math:: 
+    k_\mathrm{p} = \frac{2 \zeta \omega_\mathrm{0,PLL}}{U_\mathrm{gN}} \qquad
+    k_\mathrm{i} = \frac{\omega_\mathrm{0,PLL}^2}{U_\mathrm{gN}}
+
+where :math:`\zeta` is the damping factor, :math:`\omega_\mathrm{0,PLL}` is the natural frequency of the PLL, and :math:`U_\mathrm{gN}` is the nominal grid voltage amplitude.
+
+More details on the control methods used can be found in [#Kau1997]_.
+
+This controller is implemented in the class :class:`motulator.grid.control.PLL`.
+
+Power Synchronization
```

Comment:
This section will be modified as it is now incorrect since the addition of observer-based GFM